controls/forms.py

```diff
@@ -55,6 +55,19 @@ class Meta:
             "poam_group": "Group"
         }
 
+class ElementForm(ModelForm):
+
+    class Meta:
+        model = Element
+        fields = ['name', 'full_name', 'description', 'element_type']
```

Comment:
For element_type it will always end up as system_element as the value going through since this is the element form. I suggest just adding a hiddeninput initial value which will provide a default for element_type. `    element_type = forms.CharField(widget=forms.HiddenInput(), initial='system_element')
`

Reply:
Fixed.